gateways/stripe/stripe_vars.php

```diff
@@ -55,3 +76,40 @@ function espresso_display_stripe($payment_data) {
 }
 
 add_action('action_hook_espresso_display_onsite_payment_gateway', 'espresso_display_stripe');
+
+
+/**
+ * Gets the number of decimal places Stripe expects a currency to have.
+ * See https://stripe.com/docs/currencies#charge-currencies for the list.
+ *
+ * @param string $currency Accepted currency.
+ * @return int
+ */
+function get_stripe_decimal_places($currency = '')
```

Comment:
should add `ee_`... we mostly copied this function from woocommerce, so likelyhood of other code having the same function name is high.

Reply:
or prefix with `espresso_` since that's the naming convention for EE3 function names